foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -158,6 +158,11 @@ def prioritize_salmon_jobs(jobs: List) -> List:
     prioritized_jobs = []
     for job in jobs:
         try:
+            if job.pipeline_applied == ProcessorPipeline.SMASHER.value \
+               or job.pipeline_applied == ProcessorPipeline.JANITOR.value \
+               or job.pipeline_applied == ProcessorPipeline.QN_REFERENCE.value:
```

Comment:
It might be nice to have this as something more maintainable, as this logic might be used in the Smasher as well?